docs/content/Details/Filters/JsonTrackReader.rst

```diff
@@ -22,10 +22,9 @@ A JSON stream which contains a list with JSON objects. The JSON objects are havi
       - decimal number
       - describes the heading
       - if available
-   - speed [km/h]
+   - speed [m/s]
```

Comment:
- You would need to change the example code below as well (meaningful values).
- As you already has made non-backward compatible changes, you may also rename the fields (in code and doc) `pos_time` to `time` and `speed` to `velocity` as you initially did ;)

Not sure yet, if we should touch existing classes... but I think the JSON reader was never broadly used? So I would accept this.